resources/lang/de/settings.php

```diff
@@ -0,0 +1,814 @@
+        ],
+        'alert' => [
+            'ack_until_clear' => [
+                'description' => 'Standard Meldung bis der Alarm abklingt',
+                'help' => 'Standard Meldung bis der Alarm abklingt'
+            ],
+            'admins' => [
+                'description' => 'Melde Alarme den Administratoren',
+                'help' => 'Alarmiere Administratoren'
+            ],
+            'default_copy' => [
+                'description' => 'Sende alle E-Mail Alarme als Kopie zum Standard Kontakt',
+                'help' => 'Sende alle E-Mail Alarme als Kopie zum Standard Kontakt'
+            ],
+            'default_if_none' => [
+                'description' => 'Kann nicht auf der Webseite konfiguriert werden?',
+                'help' => 'Sende E-Mail zum Standard Kontakt wenn keine anderen Kontakte gefunden werden'
+            ],
+            'default_mail' => [
+                'description' => 'Standard Kontakt',
+                'help' => 'Der Standard E-Mail Kontakt'
+            ],
+            'default_only' => [
+                'description' => 'Sende Alarme nur zum Standard Kontakt',
+                'help' => 'Alarmiere nur den Standard E-Mail Kontakt'
+            ],
+            'disable' => [
+                'description' => 'Deaktivere Alarmierungen',
+                'help' => 'Stoppe Generierung von Alarmen'
+            ],
+            'fixed-contacts' => [
+                'description' => 'Update der Kontakt E-Mail Adresse werden nicht berücksichtigt',
+                'help' => 'Wenn aktiviert werden jegliche Änderungen in sysContact oder Benutzer E-Mails nicht berücksichtigt solange Alarme aktiv sind'
+            ],
+            'globals' => [
+                'description' => 'Melde Alarme zu Benutzer welche nur Leserechte haben',
+                'help' => 'Alarmiere Administratoren welche nur Leserechte haben'
+            ],
+            'syscontact' => [
+                'description' => 'Melde Alarme an sysContact',
+                'help' => 'Sende AlarmierungsE-Mail an den SNMP sysContact'
+            ],
+            'transports' => [
+                'mail' => [
+                    'description' => 'Aktiviere Alarmierungen via E-Mail',
+                    'help' => 'Benachrichtige über Alarme via E-Mail'
+                ]
+            ],
+            'tolerance_window' => [
+                'description' => 'Toleranzfenster für cron',
+                'help' => 'Toleranzfenster in Sekunden'
+            ],
+            'users' => [
+                'description' => 'Benachrichtige normale Benutzer über Alarme',
+                'help' => 'Alarmiere normale Benutzer'
+            ]
+        ],
+        'alert_log_purge' => [
+            'description' => 'Entferne Alarm Logeinträge älter als',
+            'help' => 'Bereinigung wirde erledigt von daily.sh',
```

Comment:
`s/wirde/wird` - Bereinigung sounds a bit wrong in this context but for lack of better alternative it should be fine

Reply:
changed